src/aap_eda/core/utils/credentials.py

```diff
@@ -245,17 +253,15 @@ def validate_injectors(schema: dict, injectors: dict) -> dict:
     if not isinstance(injectors, dict):
         errors.append("Injectors must be in Key-Value pairs format")
 
-    if not any(
-        support_key in injectors.keys()
-        for support_key in SUPPORTED_KEYS_IN_INJECTORS
-    ):
+    injector_keys = set(injectors.keys())
+    if not injector_keys or not SUPPORTED_KEYS_IN_INJECTORS >= injector_keys:
```

Comment:
@mkanoor we can compare `SUPPORTED_KEYS_IN_INJECTORS >= injector_keys`? LHS is a dict, and RHS is a set?